compiler/rustc_codegen_llvm/src/gotoc/metadata.rs

```diff
@@ -12,26 +12,14 @@ use rustc_data_structures::owning_ref::OwningRef;
 use rustc_data_structures::rustc_erase_owner;
 use rustc_data_structures::stable_map::FxHashMap;
 use rustc_data_structures::sync::MetadataRef;
-use rustc_hir::def_id::{DefId, LOCAL_CRATE};
+use rustc_hir::def_id::LOCAL_CRATE;
 use rustc_middle::middle::cstore::MetadataLoader;
 use rustc_middle::mir::interpret::Allocation;
-use rustc_middle::mir::{HasLocalDecls, Local, Operand, Place, Rvalue};
 use rustc_middle::ty::layout::{HasParamEnv, HasTyCtxt, TyAndLayout};
-use rustc_middle::ty::print::with_no_trimmed_paths;
-use rustc_middle::ty::{self, Instance, Ty, TyCtxt, TypeFoldable};
+use rustc_middle::ty::{self, Instance, Ty, TyCtxt};
 use rustc_target::abi::{HasDataLayout, LayoutOf, TargetDataLayout};
 use rustc_target::spec::Target;
-use std::iter;
 use std::path::Path;
-use tracing::debug;
-
-// #[derive(RustcEncodable, RustcDecodable)]
-pub struct GotocCodegenResult {
-    pub symtab: SymbolTable,
-    pub crate_name: rustc_span::Symbol,
-}
-
-pub struct GotocMetadataLoader();
 
```

Comment:
Could we add header to this comment to explain roughly what's here? I know `metadata.rs` is a name from elsewhere in Rust, but it's not very descriptive.